### ISSUE TITLE
Imagine engine dependency build issue - libarchive

### ISSUE THREAD

**Issue by NE1WeKnow** [Rakashazi/emu-ex-plus-alpha#90]
I have followed instructions for environment setup here
[https://github.com/Rakashazi/emu-ex-plus-alpha/blob/master/imagine/doc/INSTALL](url)

Running build using

`bash makeAll-ios-arm64.sh TAR=gnutar`

Build is incomplete as below hence I can't continue:

`Extracting libarchive...
tar -mxJf libarchive-3.2.1.tar.xz -C /tmp/imagine-bundle/libarchive/libarchive-3.2.1/..
patch -d /tmp/imagine-bundle/libarchive/libarchive-3.2.1 -p1 < libarchive-3.2.0-entry-crc32.patch #` 

and then further down

`configure.ac:135:` error: possibly undefined macro: AC_MSG_FAILURE
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:295: error: possibly undefined macro: AC_CHECK_LIB`

**Comment by thededman** [comment]
Do you have all the MACPORTS installed? www.macports.com
Install MacPort per the instruction from MacPorts I will not cover here. 
Open Terminal, and **sudo port selfupdate** let this complete. Now type **sudo port install coreutils libtool pkgconfig gnutar autoconf** this will take a few minutes. **This is need to complete the compile.** 

And insure xcode command line tools is installed 
xcode-select --install

**Comment by NE1WeKnow** [comment]
Thanks for the help, David (I'm guessing it's David from the output above!)

I'll be honest - I don't know what I'm doing, but I'm not your usual dumb user trying to do something when I'm way out of my depth.  I am a pretty technically savvy person and very good at reading through instructions / tutorials and finding solutions. 

I don't have a Mac just like you don't, so everything I've picked up has been a bit of a struggle.  However, I've previously used my virtual environment and Xcode to build other emulators.

I ran through the Macports update again as you described but it gets a long way through before coming unstuck at the same place.

Is it not the "right thing" to upload a DEB or unsigned IPA of latest version for C64.emu?  I have version 1.5.21 right now but having problems with signing it and was only able to push it to my iphone using a free developer account.  I believe it will be unusable in a few days time.
